_includes/footer.html

```diff
@@ -5,78 +5,26 @@
-                    <div class="title">
-                        Column 2
-                    </div>
                     <ul class="links">
-                        <li><a href="">About Us</a></li>
-                        <li><a href="">This is a super long link</a></li>
-                        <li><a href="">Test</a></li>
-                        <li><a href="">Test</a></li>
+                        <li><a href="/about/">About Us</a></li>
+                        <li><a href="/components/">Components</a></li>
+                        <li><a href="/patterns/">Patterns</a></li>
+                        <li><a href="/templates/">Templates</a></li>
+                        <li><a href="https://customiser.designsystem.gov.sg/">Theme customiser</a></li>
+                        
                     </ul>
                 </div>
+                
                 <div class="col-xxl-2 col-md-4 mb-3">
-                    <div class="title">
-                        Column 3
-                    </div>
-                    <ul class="links">
-                        <li><a href="">About Us</a></li>
-                        <li><a href="">This is a super long link</a></li>
-                        <li><a href="">Test</a></li>
-                        <li><a href="">Test</a></li>
-                    </ul>
-                </div>
-              <div class="col-xxl-2 col-md-4 mb-3">
-                    <div class="title">
-                        Column 4
-                    </div>
-                    <ul class="links">
-                        <li><a href="">About Us</a></li>
-                        <li><a href="">This is a super long link</a></li>
-                        <li><a href="">Test</a></li>
-                        <li><a href="">Test</a></li>
-                    </ul>
-                </div>
-              <div class="col-xxl-2 col-md-4 mb-3">
-                    <div class="title">
-                        Column 5
-                    </div>
-                    <ul class="links">
-                        <li><a href="">About Us</a></li>
-                        <li><a href="">This is a super long link</a></li>
-                        <li><a href="">Test</a></li>
-                        <li><a href="">Test</a></li>
-                    </ul>
-                </div>
-              <div class="col-xxl-2 col-md-4 mb-3">
-                    <div class="title">
-                        Column 6
-                    </div>
                     <ul class="links">
-                        <li><a href="">About Us</a></li>
-                        <li><a href="">This is a super long link</a></li>
-                        <li><a href="">Test</a></li>
-                        <li><a href="">Test</a></li>
+                        <li><a href="/faqs/">FAQs</a></li>
+                        <li><a href="https://www.isomer.gov.sg" target="_blank" rel="noopener noreferrer">Isomer</a></li>
                     </ul>
```

Comment:
Add in https://www.developer.tech.gov.sg

Reply:
add in [eae3070](https://github.com/GovTechSG/sgds/pull/367/commits/eae307099c363ff38061e4e36b806b2e9057a13e)